test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Csv.Tests.ps1

```diff
@@ -43,81 +45,154 @@ Describe "Export-Csv" -Tags "CI" {
         $expected = @("#TYPE System.String", "`"Length`"", $first , $second, $third)
 
         for ($i = 0; $i -lt $expected.Count; $i++) {
-            $(Get-Content $testCsv)[$i] | Should -Be $expected[$i]
+            $(Get-Content -Path $testCsv)[$i] | Should -Be $expected[$i]
         }
     }
 
     It "Does not include type information by default" {
         $testObject | Export-Csv -Path $testCsv
+        $results = Get-Content -Path $testCsv
 
-        $(Get-Content $testCsv)[0] | Should -Not -Match ([regex]::Escape("System.String"))
-        $(Get-Content $testCsv)[0] | Should -Not -Match ([regex]::Escape("#TYPE"))
+        $results[0] | Should -Not -Match ([regex]::Escape("System.String"))
+        $results[0] | Should -Not -Match ([regex]::Escape("#TYPE"))
     }
 
     It "Does not include type information with -NoTypeInformation" {
         $testObject | Export-Csv -Path $testCsv -NoTypeInformation
+        $results = Get-Content -Path $testCsv
 
-        $(Get-Content $testCsv)[0] | Should -Not -Match ([regex]::Escape("System.String"))
-        $(Get-Content $testCsv)[0] | Should -Not -Match ([regex]::Escape("#TYPE"))
+        $results[0] | Should -Not -Match ([regex]::Escape("System.String"))
+        $results[0] | Should -Not -Match ([regex]::Escape("#TYPE"))
     }
 
     It "Includes type information when -IncludeTypeInformation is supplied" {
         $testObject | Export-Csv -Path $testCsv -IncludeTypeInformation
+        $results = Get-Content -Path $testCsv
 
-        $(Get-Content $testCsv)[0] | Should -BeExactly "#TYPE System.String"
+        $results[0] | Should -BeExactly "#TYPE System.String"
     }
 
     It "Does not support -IncludeTypeInformation and -NoTypeInformation at the same time" {
-        { $testObject | Export-Csv -Path $testCsv -IncludeTypeInformation -NoTypeInformation } | 
-            ShouldBeErrorId "CannotSpecifyIncludeTypeInformationAndNoTypeInformation,Microsoft.PowerShell.Commands.ExportCsvCommand"
+        { $testObject | Export-Csv -Path $testCsv -IncludeTypeInformation -NoTypeInformation } | Should -Throw -ErrorId "CannotSpecifyIncludeTypeInformationAndNoTypeInformation,Microsoft.PowerShell.Commands.ExportCsvCommand"
     }
-}
 
-Describe "Export-Csv DRT Unit Tests" -Tags "CI" {
-    BeforeAll {
-        $filePath = Join-Path $TestDrive -ChildPath "test.csv"
-        $newLine = [environment]::NewLine
+    It "Should support -LiteralPath parameter" {
+        $testObject | Export-Csv -LiteralPath $testCsv
+        $results = Import-Csv -Path  $testCsv
+
+        $results | Should -HaveCount 3
+    }
+
+    It "Should overwrite file without -NoClobber parameter" {
+        $P1 | Export-Csv -Path $testCsv
+        $P2 | Export-Csv -Path $testCsv
+        $results = Import-Csv -Path $testCsv
+
+        $results.P2 | Should -BeExactly "second"
+        $property = $results.PSObject.Properties.Name
+        $property | Should -BeExactly "P2"
```

Comment:
Seems it is a duplicate line 91 and should be removed- the line failed if we haven't "P2" property and we never reach line 93. Below too.

Reply:
Maybe better remove 92 and 93, since line 91 covers both property name and value?

Reply:
Yes.

Reply:
Fixed.